facebook/__init__.py

```diff
@@ -84,7 +84,7 @@ def __init__(
         app_secret=None,
     ):
         # The default version is only used if the version kwarg does not exist.
-        default_version = VALID_API_VERSIONS[0]
+        default_version = VALID_API_VERSIONS[8]
```

Comment:
He's talkin about this line. This is a breaking change, so leave this at 0 but open a new PR if you want to change it

Reply:
ok thanks i will change this and open a new PR i have been busy sorry

Reply:
sorry i have been busy, i have reverted it back will take a look on the code and update when i have time

Reply:
No worries, just wanted to make sure you could merge it 🙏